src/components/SchemasListing/index.svelte

```diff
@@ -8,26 +8,41 @@
       {schema}</a>
   </dt>
   <dd>
+   <Definitions select_instance={select_instance} definitions="{schema_definitions[schema]}"> </Definitions>
   </dd>
 {/each}
 </dl>
   </div>
 
 <script>
-    export let schemas = {};
+  export let schemas = {};
   export let selected_instance_url;
 
   import {
     createEventDispatcher
   } from 'svelte';
 
+  import Definitions from "../Instance/Definitions/index.svelte"
```

Comment:
`Definitions` is used by the SchemasListing component. I'd make more sense if it was located under it, rather than under Instance.